packages/meditrak-server/src/tests/apiV2/dashboardVisualisations/GETDashboardVisualisations.test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { setupTest } from '@tupaia/database';
+import { camelKeys } from '@tupaia/utils';
+
+import { expect } from 'chai';
+import { resetTestData, TestableApp } from '../../testUtilities';
+import { findTestRecordByCode, TEST_SETUP } from './dashboardVisualisations.fixtures';
+
+describe('GET dashboard visualisations', () => {
+  const expectError = (response, error) => {
```

Comment:
There is! We define custom `jest` matchers at [jest.setup.js](https://github.com/beyondessential/tupaia/blob/dev/jest.setup.js). It would be nice to use them in the `expect` chain, eg:

```js
expect(response).toFailWith(error);
expect(response).toSucceedWith(payload);
```

I didn't do that now because we use `chai` in `meditrak-server` tests. I guess there would be a way to define custom assertions there too, but I am not familiar with it. That being said, I can probably turn those in reusable functions. Thanks for pointing this out!